servers/audio_server.cpp

```diff
@@ -973,7 +973,15 @@ void AudioServer::init() {
 	channel_disable_threshold_db = GLOBAL_DEF_RST("audio/channel_disable_threshold_db", -60.0);
 	channel_disable_frames = float(GLOBAL_DEF_RST("audio/channel_disable_time", 2.0)) * get_mix_rate();
 	ProjectSettings::get_singleton()->set_custom_property_info("audio/channel_disable_time", PropertyInfo(Variant::FLOAT, "audio/channel_disable_time", PROPERTY_HINT_RANGE, "0,5,0.01,or_greater"));
-	buffer_size = 1024; //hardcoded for now
+	AudioDriver *driver = nullptr;
+	if (AudioDriver::get_singleton())
+		driver = AudioDriver::get_singleton();
+
+	buffer_size = driver->get_mix_buffer_size();
```

Comment:
This can give a null pointer dereference.

It should be:
```suggestion
	if (AudioDriver::get_singleton())
		buffer_size = driver->get_mix_buffer_size();
```

Reply:
> Could you squash commits into one?

Will do along with with fixing the null pointer dereference